src/components/GraphqlClient/GraphqlView.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import GraphiQL from 'graphiql';
+import { withStyles } from '@material-ui/core';
+import fetch from 'isomorphic-fetch';
+import env from '../../utils/env';
+
+const BACKEND = env.REACT_APP_BACKEND_API;
+
+function graphQLFetcher(graphQLParams) {
+  return fetch(BACKEND, {
+    method: 'post',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(graphQLParams),
+  }).then((response) => response.json());
+}
+
+const GraphqlView = ({ classes }) => (<div className={classes.grapqhQlContainer}><GraphiQL editorTheme="solarized light" fetcher={graphQLFetcher} /></div>);
```

Comment:
Codacy has a fix for the issue: [Function component is not a function expression](https://app.codacy.com/gh/CBIIT/bento-icdc-frontend/pullRequest?prid=8552752)
```suggestion
var GraphqlView = function({ classes }) {
```